specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-06-01/netapp.json

```diff
@@ -207,6 +207,43 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/quotaLimits": {
```

Comment:
>quotaLimits [](http://example.com/codeflow?start=85&length=11)

Per API contract this API must return a collection GET and expose another API to return a single item (singleton), see here: [https://github.com/Azure/azure-resource-manager-rpc/blob/770933ab2af1ece1eba2835f7a03bbfde47bb793/v1.0/common-api-contracts.md#singleton-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/770933ab2af1ece1eba2835f7a03bbfde47bb793/v1.0/common-api-contracts.md#singleton-resources)

Reply:
Looks like this will require a service side change.

Reply:
@jorgecotillo We have updated the spec with proposed implementation to address this. Can you take another look?